Capri.Web.ViewModels/Promoter/PromoterUpdate.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Capri.Web.ViewModels.Promoter
+{
+    public class PromoterUpdate
```

Comment:
I'm thinking, that maybe it will be better to update even more data (like email, etc).